sdk/harambe/settings.py

```diff
@@ -0,0 +1,14 @@
+from functools import lru_cache
+
+# from pydantic import SecretStr
```

Comment:
nit: remove

Reply:
Yea, I was planning to remove this.
Actually its recommended to use SecretStr but it masks it even when we actually need so, I was trying to figure it out.